### PR TITLE
Dev to master

### PR DESCRIPTION
**Pull request by aaronpcz**
After dev changes applied to master, dev can be brought up to date and we have unified our branches again.

Ideally this would be dev only and master simply keeps up to date with dev at regular intervals but as a number of PRs were made and missed that directly went to master, we have this as a final resort.  The end goal results in the same outcome, but a bit messy.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +85,8 @@
     "react-image": "^1.3.1",
     "react-imageloader": "2.1.0",
     "react-measure": "^1.4.7",
-    "react-monaco-editor": "^0.7.3",
+    "monaco-editor": "^0.9.0",
```
Comment (aaronpcz) [inline package.json]: I'm not sure why this dependency is in the package json.  I need to track where it has come from.

We should not be pulling in the monaco-editor, I'm sure the react-monaco-editor package should....

Comment (aaronpcz) [reply package.json]: will debug

Comment (Stormgun) [reply package.json]: For some reason with react-monaco-editor it does not install the monaco-editor dependency for itself. 
The npm installation logs gave me a message of monaco-editor@0.9.0 expected

Comment (aaronpcz) [reply package.json]: it's cool it brings in the monaco editor as a peer dep - in the 0.9.x range, so we good for now.

**Comment by aaronpcz** [comment]
Looks like we are good - I do not have admin writes to force merge this so a quick approval review would be great @Stormgun.

After approval I'll hold back on the merge until we have a free period later in the day with no users.

**Pull request merged**